collectors/StatsCollector.py

```diff
@@ -53,15 +50,16 @@ def collect(self):
                 if statkey in metrics and metric_data is not None:
                     metrics[statkey]['gauge'].add_metric(labels=labels, value=metric_data)
                 else:
-                    no_match_in_config.add(statkey)
-
-        if list(no_match_in_config):
-            logger.warning(f'Skipped keys, due to no match with config in {self.name}: {list(no_match_in_config)} '
-                           f'<-- compare with collector_config')
+                    new_metric_suffix = re.sub('[^a-zA-Z/s0-9\n.]', '_', statkey)
```

Comment:
Can we add a comment here? I am sure I forget why we are doing this. Or do some renaming, the actual `no_match_in_config` was self-explanatory.